src/Output.tsx

```diff
@@ -62,7 +63,7 @@ export function GraphOutput<S extends Schema, K extends keyof S>(
 				dominantBaseline="middle"
 				fontSize={fontSize}
 			>
-				{props.output}
+				{kinds[node.kind].outputs[output].label}
```

Comment:
Nit: Same as in Input.tsx. Maybe this could be done in the parent component? But not really necessary at the moment. Let me know what you think.